frontend/src/lib/components/icons/AddToListIcon.svelte

```diff
@@ -0,0 +1,20 @@
+<script lang="ts">
```

Comment:
All the icons contain the same code. Should it be better to create a generic Icon and pass the name of the icon we want to render?. We could use a types list to provide some intellisense and allow people to know the available icons

Reply:
There are some drawbacks to that, but we can go either way. See discussion in the new PR #234 

The code is not totally identical for each of them, though. What varies are:

- `title`
- `viewBox` (although it's the same in most of them)
- whether to apply `color` to `fill` or `stroke` (although in the current batch, it's always fill)
- the SVG contents, naturally

This approach was modelled after https://github.com/ramiroaisen/svelte-material-icons/tree/master/svelte-material-icons